tests/e2e/policy_driven_vol_allocation.go

```diff
@@ -162,7 +162,8 @@ var _ = ginkgo.Describe("[vol-allocation] Policy driven volume space allocation
 			10	Deleted the SPBM polices created in step 1
 	*/
 	ginkgo.It("[csi-block-vanilla][csi-block-vanilla-parallelized][csi-guest][csi-supervisor]"+
-		"[csi-wcp-vsan-direct] Verify Thin, EZT, LZT volume creation via SPBM policies", func() {
+		"[csi-wcp-vsan-direct] Verify Thin, EZT, LZT volume creation via SPBM "+
```

Comment:
Please put block label as well for all policy driven allocation testcases as this is only applicable for block volumes.

Reply:
Should this be P0?